envknob/envknob.go

```diff
@@ -142,3 +142,6 @@ func LookupInt(envVar string) (v int, ok bool) {
 // UseWIPCode is whether TAILSCALE_USE_WIP_CODE is set to permit use
 // of Work-In-Progress code.
 func UseWIPCode() bool { return Bool("TAILSCALE_USE_WIP_CODE") }
+
+// CanSSHD is whether the Tailscale SSH server is allowed to run.
```

Comment:
Can you clarify what happens if ssh is already enabled?

Reply:
It doesn't start.

Reply:
It's not allowed to run. I think the comment's pretty clear already.

Reply:
That is, it says nothing about state transitions here at all.

Reply:
PTAL